eng/config.json

```diff
@@ -38,7 +38,7 @@
         },
         {
             "Name": "azdatalake",
-            "CoverageGoal": 0.60
+            "CoverageGoal": 0.00
```

Comment:
According to the test results, current CC is 26.17%. Maybe set this to something like 0.25?

Reply:
Fair, changed to 0.25 for now